CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Increased timeout on n3h spawn and wait for `#P2P-READY#` message [#1276](https://github.com/holochain/holochain-rust/pull/1276).
+- Default N3H mode as set when spawned by the conductor got set to "REAL". [#1282](https://github.com/holochain/holochain-rust/pull/1282)
```

Comment:
Could we take this opportunity to clarify internally and externally what those nodes actually are? I personally don’t even know, in clear terms

Reply:
REAL is a big update to HACK mode.
HACK is going to dissapear shortly. 
Consider REAL to be HACK mode's new name.
I needed to keep both around to make sure things didn't break.

REAL is in opposition to MOCK mode.
Mock mode is where you connect multiple local nodes to one n3h in order to simulate a network over IPC.
REAL is the normal n3h mode that gossips over the network.
Maybe renamed it to NORMAL?

Is that clear enough for you?

Reply:
Yep

Reply:
@Connoropolous @ddd-mtl can it be a small followup to do a rename and document this?